packages/cursorless-engine/src/spokenForms/defaultSpokenFormMapCore.ts

```diff
@@ -107,6 +107,7 @@ export const defaultSpokenFormMapCore: DefaultSpokenFormMapDefinition = {
     textFragment: isPrivate("text fragment"),
     disqualifyDelimiter: isPrivate("disqualify delimiter"),
     pairDelimiter: isPrivate("pair delimiter"),
+    interior: isPrivate("inside"),
```

Comment:
do we want to use a different default name for this? I don't feel strongly, but might be useful to differentiate from the modifier "inside"

Reply:
I think also used for scope visualizer during dev. I don't feel strongly, but I think if you enabled the scope so you could use it for scope visualizer in dev, you'd end up getting "inside" the scope clashing with "inside" the modifier? So yeah maybe "interior" would be fine?